packages/nextjs-plugin/src/custom-webpack-loader.js

```diff
@@ -9,8 +9,14 @@
 
 const PLUGIN_NAME = 'stylex';
 
-module.exports = function stylexLoader(inputCode) {
+module.exports = function stylexLoader(inputCode, inputSourceMap) {
   const callback = this.async();
+
+  // bail out early if the input doesn't contain "stylex"
+  if (!inputCode.includes('stylex')) {
```

Comment:
If an alias is used, this will break. Instead, you can read the `inputSources` option (defaulting to `@stylexjs/stylex`) and use that for checking instead.

Reply:
Sure, let me fix that.